modules/database/sqlite/constants.php

```diff
@@ -6,12 +6,23 @@
  * @package performance-lab
  */
 
+// Backwards-compatibility.
+if ( ! defined( 'PERFLAB_SQLITE_DB_DROPIN_VERSION' ) && defined( 'SQLITE_DB_DROPIN_VERSION' ) ) {
+	define( 'PERFLAB_SQLITE_DB_DROPIN_VERSION', SQLITE_DB_DROPIN_VERSION );
+}
```

Comment:
See my other comment, we also need to ensure backward compatibility the other direction, especially as I'm assuming `SQLITE_DB_DROPIN_VERSION` will be our main constant going forward.
```suggestion
}
if ( ! defined( 'SQLITE_DB_DROPIN_VERSION' ) && defined( 'PERFLAB_SQLITE_DB_DROPIN_VERSION' ) ) {
	define( 'SQLITE_DB_DROPIN_VERSION', PERFLAB_SQLITE_DB_DROPIN_VERSION );
}
```